yacht/__init__.py

```diff
@@ -10,63 +10,84 @@
 from . import sketch_sample
 from .utils import __version__
 
+
 def main():
-    parser = argparse.ArgumentParser(prog='yacht')
-    subparsers = parser.add_subparsers(dest='command')
+    parser = argparse.ArgumentParser(prog="yacht")
+    subparsers = parser.add_subparsers(dest="command")
 
     # Train command
-    train_parser = subparsers.add_parser('train', description='Pre-process the reference genomes')
+    train_parser = subparsers.add_parser(
```

Comment:
Why the odd formatting of the arguments to `add_parser`? This "lots of line breaks" approach appears inconsistently applied and counter to PEP8's recommendation for artificial line breaks only after 79 characters

Reply:
This linter tries to divide the line if line is longer then 88 characters